robot/basestation/app.py

```diff
@@ -245,6 +247,36 @@ def capture_image():
 
     return jsonify(msg=msg)
 
+# This is called on if ffmpeg isn't already recording when the record toggle button is clicked
+@app.route("/start_recording_feed/<stream>", methods=["POST", "GET"])
+def start_recording_feed(stream):
+    date = datetime.datetime.now().strftime("%I:%M:%S_%B_%d_%Y")
+    # Name of file to be saved
+    name = stream + '_' + date
+    # Get stream url
+    stream_url = "http://" + fetch_ros_master_ip() + ":8080/stream?topic=/cv_camera/image_raw"
+    # Message to print to log
```

Comment:
I would recommend changing your variables to something more descriptive and remove the comments. 

example

`formatted_date = ...`

`name_of_file = ...`

`recording_log_msg`

Right now many comments must be read because the variable names are not descriptive enough